Mergin/project_status_dialog.py

```diff
@@ -213,3 +221,28 @@ def show_changes(self):
         self.ui.messageBar.pushMessage("Mergin", "No changes found in the project layers.", Qgis.Info)
         dlg_diff_viewer.show()
         dlg_diff_viewer.exec_()
+
+    def link_clicked(self, url):
+        if self.module is None:
+            self.ui.messageBar.pushMessage("Mergin", f"Repair module not found", Qgis.Warning)
+            return
+
+        function_name = url.toString().strip("#")
+        if hasattr(self.module, function_name):
+            ok, msg = getattr(self.module, function_name)(self.mp)
```

Comment:
I would prefer to make this more explicit - e.g. if the link was "#fix_datum_shift_grids" then we will directly call `fix_datum_shift_grids()` - otherwise it's difficult to know what are the supported fixes (and by looking for references to the function in IDE it may look like the `fix_datum_shift_grids()` is unused...